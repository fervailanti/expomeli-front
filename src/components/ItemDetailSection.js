import React from 'react'

const ItemDetailSection = ({ title, description }) => (
  <div className="detail-section">
    <p className="title">{title}</p>
    <p className="description">{description}</p>
  </div>
)

export default ItemDetailSection
