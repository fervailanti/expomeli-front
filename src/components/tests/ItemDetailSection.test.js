import React from 'react'
import ItemDetailSection from '../ItemDetailSection'
import { render } from '@testing-library/react'

it('ItemDetailSection component should render correctly', () => {
  const tree = render(
    <ItemDetailSection
      title="DescripciÃ³n del producto"
      description="lorem ipsum lorem ipsum lorem ipsum lorem"
    />
  )
  expect(tree.container).toMatchSnapshot()
})
