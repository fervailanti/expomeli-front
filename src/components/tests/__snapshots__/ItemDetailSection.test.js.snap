// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemDetailSection component should render correctly 1`] = `
<div>
  <div
    class="detail-section"
  >
    <p
      class="title"
    >
      Descripci√≥n del producto
    </p>
    <p
      class="description"
    >
      lorem ipsum lorem ipsum lorem ipsum lorem
    </p>
  </div>
</div>
`;
